packages/graphql/src/translate/create-auth-and-params.test.ts

```diff
@@ -94,6 +94,92 @@ describe("createAuthAndParams", () => {
                 this_auth_allow0_id: sub,
             });
         });
+
+        test("should combine roles with where across rules", () => {
+            const idField = {
+                fieldName: "id",
+                typeMeta: {
+                    name: "ID",
+                    array: false,
+                    required: false,
+                    pretty: "String",
+                    input: {
+                        where: {
+                            type: "String",
+                            pretty: "String",
+                        },
+                        create: {
+                            type: "String",
+                            pretty: "String",
+                        },
+                        update: {
+                            type: "String",
+                            pretty: "String",
+                        },
+                    },
+                },
+                otherDirectives: [],
+                arguments: [],
+            };
+
+            const node = new NodeBuilder({
+                name: "Movie",
+                relationFields: [],
+                cypherFields: [],
+                enumFields: [],
+                scalarFields: [],
+                primitiveFields: [idField],
+                temporalFields: [],
+                interfaceFields: [],
+                objectFields: [],
+                pointFields: [],
```

Comment:
Most of these empty array fields are already set by NodeBuilder class, so we should be able to remove all default fields:
```suggestion
                primitiveFields: [idField],
```